models/api.model.js

```diff
@@ -103,3 +103,12 @@ exports.deleteCommentById = async id => {
     throw new Error("Failed to delete comment");
   }
 };
+
+exports.fetchUsers = async () => {
+  try {
+    const { rows } = await db.query(`SELECT * FROM users;`);
+    return rows;
+  } catch (err) {
+    throw new Error("Failed to get users");
```

Comment:
Not relevant here as you don't need it, but I'd recommend a Promise.reject with a msg key rather than throwing an error.